src/web/routes/application/children-dob/children-dob.js

```diff
@@ -12,13 +16,39 @@ const pageContent = ({ translate }) => ({
   aboutYourChild: translate('childrenDob.aboutYourChild')
 })
 
+const behaviourForGet = (req, res, next) => {
+  if (!req.session.hasOwnProperty('children')) {
+    req.session.children = {}
+  }
+
+  req.session.children.count = 1
```

Comment:
Shouldn't this be inside the if check? Otherwise won't it reset the count of children to 1 every get request?